superset/views/core.py

```diff
@@ -2414,7 +2413,7 @@ def validate_sql_json(
             )
 
         spec = mydb.db_engine_spec
-        validators_by_engine = get_feature_flags().get("SQL_VALIDATORS_BY_ENGINE")
+        validators_by_engine = app.config["SQL_VALIDATORS_BY_ENGINE"]
```

Comment:
It appears that SQL validation has never worked based on the `SQL_VALIDATORS_BY_ENGINE` top level config parameter, as this was referencing the feature flag map.